src/app/services/manipulate-data.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import * as models from '../models';
+
+@Injectable()
+export class ManipulateDataService {
```

Comment:
good point - we could go for 'DataHandlerService'  or 'SetterGetterService', let me know if you come up with a more descriptive name